slack_driver/requirements.txt

```diff
@@ -0,0 +1,7 @@
+credstash
```

Comment:
You should especially freeze credstash at a certain version.  I'd recommend though that you pip freeze this whole file.

Reply:
hah this always bothers me when its frozen, and when its unfrozen